src/main/java/me/totalfreedom/tfguilds/command/DisbandSubCommand.java

```diff
@@ -1,16 +1,21 @@
 package me.totalfreedom.tfguilds.command;
 
+import java.util.HashMap;
 import me.totalfreedom.tfguilds.Common;
+import me.totalfreedom.tfguilds.TFGuilds;
 import me.totalfreedom.tfguilds.guild.Guild;
 import org.apache.commons.lang.StringUtils;
 import org.bukkit.Bukkit;
 import org.bukkit.ChatColor;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
 
 public class DisbandSubCommand extends Common implements SubCommand
 {
 
+    private final HashMap<CommandSender, Boolean> confirm = new HashMap<>();
```

Comment:
It's best to use `List<CommandSender>` instead of storing the information in boolean, mainly when the value is not used.

Reply:
Thank you very much!